doc/api/path.md

```diff
@@ -332,7 +332,7 @@ path.normalize('/foo/bar//baz/asdf/quux/..')
 On Windows:
 
 ```js
-path.normalize('C:\\temp\\\\foo\\bar\\..\\');
+path.normalize('C:\\temp\\\\foo\\bar\\..\\')
```

Comment:
There are sections where all semicolons are present, there are ones where all are omitted. In the most cases when examples are intended to run in REPL all the semicolon are omitted. In this section one is present and one is omitted. I've just tried to make it consistent and similar to others.